src/NServiceBus.AcceptanceTests/Mutators/When_using_outgoing_tm_mutator.cs

```diff
@@ -39,7 +39,6 @@ public Task MutateOutgoing(MutateOutgoingTransportMessageContext context)
                 {
                     context.OutgoingHeaders["HeaderSetByMutator"] = "some value";
                     context.OutgoingHeaders[Headers.EnclosedMessageTypes] = typeof(MessageThatMutatorChangesTo).FullName;
-                    context.OutgoingBody = Encoding.UTF8.GetBytes("<MessageThatMutatorChangesTo/>");
```

Comment:
My only concern is that this would mean that we no longer verify that the body can be changed? (but I guess the old tests didn't check that either since it worked even if we didn't change)

To fix that we would have to add a property to the new message and set that via the raw xml.

@timbussmann why would it be hard to do this? (`.Repeat.For(Serializers.Xml)` ?